.all-contributorsrc

```diff
@@ -12123,6 +12123,15 @@
       "contributions": [
         "bug"
       ]
+    },
```

Comment:
I think this is handled by a script @wackerow?

Reply:
Should I take it out and push the changes?